midi2audio.py

```diff
@@ -36,17 +36,18 @@
 
 DEFAULT_SOUND_FONT = '~/.fluidsynth/default_sound_font.sf2'
 DEFAULT_SAMPLE_RATE = 44100
+DEFAULT_GAIN = 0.2
```

Comment:
Does it match the default in the fluidsynth command. If not, it would be a breaking change.

Reply:
Yes, the default is 0.2 [https://www.fluidsynth.org/api/fluidsettings.xml#synth.gain](https://www.fluidsynth.org/api/fluidsettings.xml#synth.gain)